<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAnalogVarAdd</class>
 <widget class="QDialog" name="DialogAnalogVarAdd">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>293</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Имя:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="2">
    <widget class="QLineEdit" name="lineEditName"/>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Приоритет:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="2">
    <widget class="QComboBox" name="comboBoxPriority">
     <item>
      <property name="text">
       <string>не отображать переменную</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>низкий приоритет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>стандартный приоритет</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>высокий приоритет</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Адрес:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="3">
    <widget class="QSpinBox" name="spinBoxAddress">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Нижнее значение шкалы:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="3">
    <widget class="QSpinBox" name="spinBoxScaleMin">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Верхнее значение шкалы:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="3">
    <widget class="QSpinBox" name="spinBoxScaleMax">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Аварийный предел мин:</string>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <widget class="QSpinBox" name="spinBoxAlarmMin">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Аварийный предел макс:</string>
     </property>
    </widget>
   </item>
   <item row="6" column="3">
    <widget class="QSpinBox" name="spinBoxAlarmMax">
     <property name="maximum">
      <number>65535</number>
     </property>
    </widget>
   </item>
   <item row="7" column="0" colspan="3">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>Коэффициент:</string>
     </property>
    </widget>
   </item>
   <item row="7" column="3">
    <widget class="QDoubleSpinBox" name="doubleSpinBoxCoeff">
     <property name="decimals">
      <number>3</number>
     </property>
    </widget>
   </item>
   <item row="8" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAnalogVarAdd</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAnalogVarAdd</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
